doc/releases/changelog-dev.md

```diff
@@ -249,6 +249,51 @@
 
 <h4>Capturing and representing hybrid programs</h4>
 
+* Traditional tape transforms in PennyLane can be automatically converted to work with program capture enabled.
+  [(#6922)](https://github.com/PennyLaneAI/pennylane/pull/6922)
+
+  As an example, here is a custom tape transform, working with capture enabled, that shifts every `qml.RX` gate to the end of the circuit:
+
+  ```python
+  qml.capture.enable()
+
+  @qml.transform
+  def shift_rx_to_end(tape):
+      """Transform that moves all RX gates to the end of the operations list."""
+      new_ops, rxs = [], []
+
+      for op in tape.operations:
+          if isinstance(op, qml.RX):
+              rxs.append(op)
+          else:
+                new_ops.append(op)
+
+      operations = new_ops + rxs
+      new_tape = tape.copy(operations=operations)
+      return [new_tape], lambda res: res[0]
+
```

Comment:
````suggestion
```
  A requirement for tape transforms to be compatible with program capture is to further decorate QNodes with the experimental `qml.capture.expand_plxpr_transforms` decorator.
```python
````